[2024-12-26 18:47:59,273] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_two 2024-12-26T18:46:02.161925+00:00 [queued]>
[2024-12-26 18:47:59,286] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_two 2024-12-26T18:46:02.161925+00:00 [queued]>
[2024-12-26 18:47:59,287] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-12-26 18:47:59,288] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-12-26 18:47:59,289] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-12-26 18:47:59,295] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): task_two> on 2024-12-26T18:46:02.161925+00:00
[2024-12-26 18:47:59,299] {standard_task_runner.py:52} INFO - Started process 1637 to run task
[2024-12-26 18:47:59,301] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'assignment', 'task_two', '2024-12-26T18:46:02.161925+00:00', '--job-id', '40', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmpnrvn5p11', '--error-file', '/tmp/tmpek8h035q']
[2024-12-26 18:47:59,303] {standard_task_runner.py:77} INFO - Job 40: Subtask task_two
[2024-12-26 18:47:59,334] {logging_mixin.py:104} INFO - Running <TaskInstance: assignment.task_two 2024-12-26T18:46:02.161925+00:00 [running]> on host d88713c29571
[2024-12-26 18:47:59,370] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=assignment
AIRFLOW_CTX_TASK_ID=task_two
AIRFLOW_CTX_EXECUTION_DATE=2024-12-26T18:46:02.161925+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-26T18:46:02.161925+00:00
[2024-12-26 18:47:59,377] {python.py:151} INFO - Done. Returned value was: None
[2024-12-26 18:47:59,386] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=assignment, task_id=task_two, execution_date=20241226T184602, start_date=20241226T184759, end_date=20241226T184759
[2024-12-26 18:47:59,405] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-26 18:47:59,436] {local_task_job.py:151} INFO - Task exited with return code 0
